es_es update

docanuto opened the pull request:
Includes spelling corrections.

Commit:
Add files via upload

es_es update.

BluSunrize commented:
How does this integrate with #4838?

@docanuto and @FrannDzs you might want to coordinate here, since it's two spanish translations.

ghost commented:
> How does this integrate with #4838?
> 
> @docanuto and @FrannDzs you might want to coordinate here, since it's two spanish translations.

From what I have seen the other translation is misspelled and wrong capitalized. Is unchecked.

Also, I think that Frann is argentinian. So the translation probably is more accurate es_ar.

FrannDzs commented:
Hi guys @docanuto @BluSunrize  I am the one who did the last translation of this mod including the manual from scratch. There were some things misspelled by me, but if I check your translation there are misspelled things too. For example, Wolf Pack is not translated literally and the industrial harvester is not a wheelbarrow. Anyway, I like your last translation, we could review it and take it as a base now. You improved some contexts and the truth is that they are very good.

I'm Argentinian, but I'm not looking to Argentinize, I want a more neutral Spanish, es_es is used as neutral Spanish in Spanish-speaking countries, most of the mods don't have Spanish translation in Argentina or Uruguay, for example, so we should make it as neutral as possible, that's why I use words from Spain and others from America.

How do we do to agree? :)

FrannDzs commented on src/main/resources/assets/immersiveengineering/lang/es_es.json:
```diff
@@ -1,1207 +1,1207 @@
+  "block.minecraft.banner.immersiveengineering_hammer.magenta": "Martillo magenta",
+  "block.minecraft.banner.immersiveengineering_hammer.orange": "Martillo naranja",
+  "block.minecraft.banner.immersiveengineering_hammer.white": "Martillo blanco",
+  "block.minecraft.banner.immersiveengineering_bevels.black": "Martillo negro",
```
Here is an error

FrannDzs commented:
> > How does this integrate with #4838?
> > @docanuto and @FrannDzs you might want to coordinate here, since it's two spanish translations.
> 
> From what I have seen the other translation is misspelled and wrong capitalized. Is unchecked.
> 
> Also, I think that Frann is argentinian. So the translation probably is more accurate es_ar.



> > How does this integrate with #4838?
> > @docanuto and @FrannDzs you might want to coordinate here, since it's two spanish translations.
> 
> From what I have seen the other translation is misspelled and wrong capitalized. Is unchecked.
> 
> Also, I think that Frann is argentinian. So the translation probably is more accurate es_ar.

Hey friend, I understand your effort, but notice that your translation is outdated, it is missing "strings", again, I love your translation, just that you should make a fork from me, so you can use my template that I have already added the lines, or as you like!

FrannDzs commented:
@docanuto do you have whatsapp or e-mail so we can communicate better?

ghost commented:
> @docanuto do you have whatsapp or e-mail so we can communicate better?

Im actually in the official Discord. :)

FrannDzs commented:
@docanuto we should make a mix of the two translations, you mark me errors and suggestions and I mark you the same, and from there we get an improved one, what do you think? start by making the fork of Immersive Engineering from my account, which is already updated, check my changes please, greetings.

> > @docanuto do you have whatsapp or e-mail so we can communicate better?
> 
> Im actually in the official Discord. :)

I'm nekrodamus on discord now

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update alloykiln.txt

paladini-maker committed:
Update alloys.txt

paladini-maker committed:
Update arc_furnace.txt

paladini-maker committed:
Update assembler.txt

paladini-maker committed:
Update assembler.txt

paladini-maker committed:
Update automated_workbench.txt

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update es_es.json

:)

paladini-maker committed:
Update es_es.json

Ready to merge

Commit:
Merge pull request #1 from FrannDzs/1.16.5

Update es_es.json

ghost commented:
Done. :)

paladini-maker committed:
Updated es_es.json

Corregidas algunas cosas para dar coherencia con lo escrito en el manual.

paladini-maker committed:
Updated Engineer's Manual (in progress)

Lo que está en negrita es lo traducido, lo demás está en inglés para un mejor orden en el trabajo.

FrannDzs commented:
An urgent upgrade is needed before this is merged.

Commit:
Merge pull request #3 from FrannDzs/1.16.5

Update for spanish language (urgently)
